domains/daki.json

```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "Mai1ii",
+           "email": "",
+           "discord": "1130452399094505534",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.aMXbg1gmpezfNF8y42-AA-Ha0JFiIjDqx1WGrcCIay3ttMG1ZHoeNaiII2Yb66HevGaQf3-vn2988HJAEfRLZyNgBJFz_ugT0VlBT-gTRr7mPkoD1AGd2w7BAYqcDy9LnZuLgKbsGN5BdiAYELVl-HMr_JdsGQ2oEJc6QlPA1SbDMk3zOab-_1raIAaSZCyn2iYJ6YUIhrFSD-MKoVIWcrnLBKstWIdOTRBhZX1TOKkt1LPVS6_2WnyL3W74XXm6NoRHEvkWfew0Vgyovnzwzsd-WUsrbwMolr6bb8Y9DjoR-6-edmZv2yzGJAr6ZuONLv8bMie8iTR2DgOM7ptPnA.RGLL06kUm83IRIat4TUh_w.PmcBOJSkYBKY-O45X-RIzejDhvi0fbQ97KyBIFyCyzkXrpV40UnwESgsVq4dyuLgHneaBK5qGit3aDugLqseWrIoVHccKxBRnnDtz7nPLhY.Ic9r31yNDvWGxVB5aMQEmQ"
+        },
+    
+        "record": {
+            "TXT": "_vercel.daki.is-a.dev to use daki.is-a.dev"
```

Comment:
this is not how you make a TXT record, make another file `_vercel.daki.json` and put the TXT there

also see https://is-a.dev/docs/vercel